src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -207,7 +190,7 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
                             CellRendererComponent={AttachmentCarouselCellRenderer}
                             renderItem={renderItem}
                             getItemLayout={getItemLayout}
-                            keyExtractor={(item) => item.source}
+                            keyExtractor={(item) => item.source as string}
```

Comment:
Avoid type assertion. Use same key extractor used in other files: source || reportActionID || index